caluma/caluma_workflow/tests/test_events.py

```diff
@@ -0,0 +1,35 @@
+from caluma.caluma_core.events import on
+from caluma.caluma_workflow.events import completed_case, completed_work_item
+
+from .. import models
+
+
+def test_events(db, work_item_factory, schema_executor):
+    work_item = work_item_factory(status=models.WorkItem.STATUS_READY, child_case=None)
+
+    @on(completed_work_item)
+    def complete_work_item_event_receiver(sender, work_item, **kwargs):
+        work_item.meta = {"been-there": "done that"}
+        work_item.save()
+        triggering_name_error_should_not_raise  # noqa
```

Comment:
I'd rather raise an exception manually than triggering one implicitly like this. Just use `raise Exception("this should not cause an error")` or something similar. Would also be kind of self-documenting this way :)

Reply:
Right! Done.